app/controllers/people_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.all
+  end
+
+
+    def show
+      @person = Person.find(params[:id])
+    end
+
+    def all_tasks
+      person = Person.find(params[:id])
+      @list = person.tasks
+    end
```

Comment:
Great idea to have a specific instance variable with all of a person's tasks that you can use in the views.